2020-12-30 14:01:00,137 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:00,137 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_2
2020-12-30 14:01:00,137 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:12,455 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:12,456 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_1
2020-12-30 14:01:12,456 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:12,456 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_3
2020-12-30 14:01:12,456 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:12,456 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:12,461 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:12,461 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_2
2020-12-30 14:01:12,461 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:37,570 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:37,571 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_3
2020-12-30 14:01:37,571 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:37,580 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:37,581 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_2
2020-12-30 14:01:37,581 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:37,588 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:37,588 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_1
2020-12-30 14:01:37,588 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:40,119 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:40,119 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_1
2020-12-30 14:01:40,119 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:40,122 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:40,122 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_2
2020-12-30 14:01:40,122 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:01:40,130 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:01:40,130 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_3
2020-12-30 14:01:40,130 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:09:26,656 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:09:26,656 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_2
2020-12-30 14:09:26,656 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:09:26,698 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:09:26,698 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_1
2020-12-30 14:09:26,698 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:09:26,715 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:09:26,715 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_3
2020-12-30 14:09:26,715 - CRITICAL - client.py - Не удалось подключиться к серверу 127.0.0.1:8888, конечный компьютер отверг запрос на подключение.
2020-12-30 14:11:00,487 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,487 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_2
2020-12-30 14:11:00,487 - DEBUG - client.py - Сформировано presence сообщение для пользователя Test_2
2020-12-30 14:11:00,487 - DEBUG - decos.py - Была вызвана функция create_presence с параметрами ('Test_2',), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,488 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40416), raddr=('127.0.0.1', 8888)>, {'action': 'presence', 'time': 1609326660.487731, 'user': {'account_name': 'Test_2'}}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:00,490 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,490 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_1
2020-12-30 14:11:00,490 - DEBUG - client.py - Сформировано presence сообщение для пользователя Test_1
2020-12-30 14:11:00,490 - DEBUG - decos.py - Была вызвана функция create_presence с параметрами ('Test_1',), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,491 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40418), raddr=('127.0.0.1', 8888)>, {'action': 'presence', 'time': 1609326660.4908667, 'user': {'account_name': 'Test_1'}}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:00,502 - DEBUG - decos.py - Была вызвана функция arg_parser с параметрами (), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,502 - INFO - client.py - Запущен клиент с парамертами: адрес сервера: 127.0.0.1 , порт: 8888, имя пользователя: Test_3
2020-12-30 14:11:00,502 - DEBUG - client.py - Сформировано presence сообщение для пользователя Test_3
2020-12-30 14:11:00,502 - DEBUG - decos.py - Была вызвана функция create_presence с параметрами ('Test_3',), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,502 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40420), raddr=('127.0.0.1', 8888)>, {'action': 'presence', 'time': 1609326660.5023322, 'user': {'account_name': 'Test_3'}}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:00,506 - DEBUG - decos.py - Была вызвана функция get_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40416), raddr=('127.0.0.1', 8888)>,), {}.Вызов из модуля common.utils.
2020-12-30 14:11:00,507 - DEBUG - client.py - Разбор приветственного сообщения от сервера: {'response': 200}
2020-12-30 14:11:00,507 - DEBUG - decos.py - Была вызвана функция process_response_ans с параметрами ({'response': 200},), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,507 - INFO - client.py - Установлено соединение с сервером. Ответ сервера: 200 : OK
2020-12-30 14:11:00,507 - DEBUG - client.py - Запущены процессы
2020-12-30 14:11:00,519 - DEBUG - decos.py - Была вызвана функция get_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40418), raddr=('127.0.0.1', 8888)>,), {}.Вызов из модуля common.utils.
2020-12-30 14:11:00,519 - DEBUG - client.py - Разбор приветственного сообщения от сервера: {'response': 200}
2020-12-30 14:11:00,519 - DEBUG - decos.py - Была вызвана функция process_response_ans с параметрами ({'response': 200},), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,519 - INFO - client.py - Установлено соединение с сервером. Ответ сервера: 200 : OK
2020-12-30 14:11:00,519 - DEBUG - client.py - Запущены процессы
2020-12-30 14:11:00,531 - DEBUG - decos.py - Была вызвана функция get_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40420), raddr=('127.0.0.1', 8888)>,), {}.Вызов из модуля common.utils.
2020-12-30 14:11:00,531 - DEBUG - client.py - Разбор приветственного сообщения от сервера: {'response': 200}
2020-12-30 14:11:00,531 - DEBUG - decos.py - Была вызвана функция process_response_ans с параметрами ({'response': 200},), {}.Вызов из модуля __main__.
2020-12-30 14:11:00,531 - INFO - client.py - Установлено соединение с сервером. Ответ сервера: 200 : OK
2020-12-30 14:11:00,531 - DEBUG - client.py - Запущены процессы
2020-12-30 14:11:18,494 - DEBUG - client.py - Сформирован словарь сообщения: {'action': 'message', 'from': 'Test_2', 'to': 'Test_3', 'time': 1609326678.494263, 'mess_text': 'hello'}
2020-12-30 14:11:18,494 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40416), raddr=('127.0.0.1', 8888)>, {'action': 'message', 'from': 'Test_2', 'to': 'Test_3', 'time': 1609326678.494263, 'mess_text': 'hello'}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:18,494 - INFO - client.py - Отправлено сообщение для пользователя Test_3
2020-12-30 14:11:18,553 - DEBUG - decos.py - Была вызвана функция get_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40420), raddr=('127.0.0.1', 8888)>,), {}.Вызов из модуля common.utils.
2020-12-30 14:11:18,553 - INFO - client.py - Получено сообщение от пользователя Test_2:
hello
2020-12-30 14:11:21,535 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40416), raddr=('127.0.0.1', 8888)>, {'action': 'exit', 'time': 1609326681.534873, 'account_name': 'Test_2'}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:21,535 - INFO - client.py - Завершение работы по команде пользователя.
2020-12-30 14:11:21,569 - CRITICAL - client.py - Потеряно соединение с сервером.
2020-12-30 14:11:25,079 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40418), raddr=('127.0.0.1', 8888)>, {'action': 'exit', 'time': 1609326685.0790896, 'account_name': 'Test_1'}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:25,079 - INFO - client.py - Завершение работы по команде пользователя.
2020-12-30 14:11:25,581 - CRITICAL - client.py - Потеряно соединение с сервером.
2020-12-30 14:11:28,271 - DEBUG - decos.py - Была вызвана функция send_message с параметрами (<socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 40420), raddr=('127.0.0.1', 8888)>, {'action': 'exit', 'time': 1609326688.2707338, 'account_name': 'Test_3'}), {}.Вызов из модуля common.utils.
2020-12-30 14:11:28,271 - INFO - client.py - Завершение работы по команде пользователя.
2020-12-30 14:11:28,594 - CRITICAL - client.py - Потеряно соединение с сервером.
